README.md

```diff
@@ -421,6 +421,14 @@ directory. It is the user's responsibility to review the hooks for e.g.
 possible security implications.
 
 ## Getting started
+
+If you want to use a stable version with ready-built images see the
+documentation of the
+[latest released version](https://github.com/kubernetes-sigs/node-feature-discovery/tree/v0.3.0).
```

Comment:
I would like to have something that tells to use all the repository of the latest release instead of the documentation. I think it could be less confusing for users. What do you think?

Reply:
Yeah, I think that's a good suggestion. Did you have some specific wording in mind? If not, I'll make up something 😄

Reply:
No, nothing really nice

Reply:
I made a small adjustment